helm-charts/splunk-otel-collector/ci/logs-only-values.yaml

```diff
@@ -3,5 +3,5 @@ splunkObservability:
   realm: fake-realm
   accessToken: fake-token
 
-tracesEnabled: false
-metricsEnabled: false
+  tracesEnabled: false
+  metricsEnabled: false
```

Comment:
Please don't include unrelated changes, better to submit separate PRs even for small fixes